prometheus_client/core.py

```diff
@@ -395,6 +395,14 @@ def close(self):
 def _MultiProcessValue(_pidFunc=os.getpid):
     files = {}
     values = []
+
+    # 1. use dict instead of nonlocal keyword to support python 2.7
```

Comment:
This comment isn't adding much to someone familiar with Python, I'd remove it.

Reply:
agree